src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -87,14 +95,35 @@ internal object AutoMend : Module(
             }
             paused = false
 
+            if (!shouldMend() && autoDisableComplete) {
+                MessageSendHelper.sendChatMessage("$chatName Mending completed! disabling")
+                disable()
+            }
+
+            if (shouldMend() && pauseAutoArmor && AutoArmor.isEnabled) {
+                AutoArmor.disable()
+                isAutoArmorPaused = true
+            }
+
+            if (takeOff) {
+                var minSlot = 9
+                for (i in 0..3) {
```

Comment:
`player.armorSlots`

Reply:
Do you know methods to convert Forge inventory slot to Minecraft Protocol inventory slot?
I don't know how...

Reply:
https://wiki.vg/Inventory

Reply:
if i have to manually convert armor slots to minecraft protocol i don't see the benefit of using player.armorSlots